### PR TITLE
Update russian.txt

### PR DESCRIPTION
**Pull request by Sedonya**
## Checklist

- [x] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: data/languages/russian.txt [hunk 1]
```diff
@@ -11,14 +11,15 @@
...
-#	unban		2023-03-10 21:50:50
-#	eblan		2023-08-07 14:27:00
-#	kaban		2023-9-21 19:00:00
-#	ByFox		2023-11-14 11:24:00
+#	banan		  2022-06-12 11:36:50
+#	ban			  2023-01-05 10:22:50
+#	unban		  2023-03-10 21:50:50
+#	eblan		  2023-08-07 14:27:00
+#	kaban		  2023-09-21 19:00:00
+#	ByFox		  2023-11-14 11:24:00
```
Comment (def-) [inline data/languages/russian.txt:21]: Is it possible not to do the changes to other contributors here?

**Review by def- — CHANGES_REQUESTED** [review]

File: data/languages/russian.txt (hunk 1 above)
Comment (Sedonya) [reply data/languages/russian.txt:21]: But it doesn't affect the data, does it? Okay, if it violates the ddnet custom very much, I apologize

Comment (Sedonya) [reply data/languages/russian.txt:21]: Oh crap, I just looked at this code on the github side, in my IDE the indents weren't like that at all 🤔

**Review by def- — APPROVED** [review]
Fixed it myself

**Pull request merged**